kube-runtime/src/utils/watch_ext.rs

```diff
@@ -4,6 +4,7 @@ use crate::{
 };
 use backoff::backoff::Backoff;
 
+use crate::utils::stream_subscribe::StreamSubscribe;
```

Comment:
This needs a feature gate.

I would add unstable features to the documentation build as well. Those are listed in `[package.metadata.docs.rs]` in kube/Cargo.toml. To simplify this it might be easiest if there was one super-feature called unstable_runtime that we could add to that.

Reply:
I added in. 

```toml
[features]
unstable_runtime = ["unstable_runtime_subscribe"]
unstable_runtime_subscribe = []
```

Is this the "super-feature" you envisaged, or would that live in the the `kube/cargo.toml`?

Reply:
e.g.

```toml
unstable-runtime = ["kube-runtime/unstable-runtime"]
```